### DISCUSSION TITLE
20240710發現

### DISCUSSION THREAD

**Discussion by TsuYiii** [ChiuAlfredo/options_day_data#35] [記錄]
結算日發現

**Comment by TsuYiii** [comment]
開盤漲幅不大，10點前發生發生跌幅120點，10點後整體波動劇烈並呈現上升趨勢，目前趨勢仍逐漸上升

**Comment by Will150143** [comment]
0710盤中影片
https://1drv.ms/f/c/b2afea2c4051527b/EhgeN1Z0zU9CtUHEUg7oc4sBSdfSVOtj1vMWhbAejbtpVw?e=DC39HR
